@xen-orchestra/web-core/lib/locales/en.json

```
@@ -26,7 +26,8 @@
 
   "core.select.all": "Select all",
   "core.select.none": "Select none",
-
+  "core.select.unselect": "Unselect all",
+  "core.select.n-selected": "{count} selected of {total} objects | {count} selected of {total} objects | {count} selected of {total} objects",
```

I'd suggest to add `-of` suffix in this case.

Also, you can remove unnecessary repetition.

```suggestion
  "core.select.n-selected-of": "{count} selected of {total} objects",
```